chapter-16/lib/game.css

```diff
@@ -31,4 +31,8 @@ box-shadow: -4px -7px 8px white, 4px -7px 8px white;
     content: "Paused";
     color: white;
     font-size: 24px;
+}
+
+.monster { 
+    background: purple 
```

Comment:
One tip: you can make the monster look more scary by adding a CSS background-image here:
```css
background-image: url("../a-monster/monster.png");
background-repeat: no-repeat;
```

You can download any .png file of a monster online, resize it to 26px width and add it to your /a-monster directory.